From 9c1db1dfb8ef56a652055e45203e487076771096 Mon Sep 17 00:00:00 2001
From: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Date: Fri, 16 Feb 2024 19:18:37 +0000
Subject: [PATCH] test: hdimage: depend on fdisk and sfdisk

The test for "gpt-partition-types" needs fdisk and sfdisk to be available.
The buildtime tests fails if fdisk is not installed.

Signed-off-by: Sudip Mukherjee <sudipm.mukherjee@gmail.com>

Forwarded: https://github.com/pengutronix/genimage/pull/235
Last-Update: 2024-02-16
---
 test/hdimage.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/hdimage.test b/test/hdimage.test
index c284613..967c22c 100755
--- a/test/hdimage.test
+++ b/test/hdimage.test
@@ -118,7 +118,7 @@ test_expect_success "gpt-overlap3" "
 	setup_gpt_files &&
 	test_must_fail run_genimage gpt-overlap3.config"
 
-test_expect_success "gpt-partition-types" "
+test_expect_success fdisk,sfdisk "gpt-partition-types" "
 	run_genimage gpt-partition-types.config &&
 	sfdisk_validate images/gpt-partition-types.img &&
 	sanitized_fdisk_sfdisk images/gpt-partition-types.img > gpt-partition-types.fdisk &&
-- 
2.39.2

